tests/mocks.py

```diff
@@ -148,3 +347,33 @@ def get_mock_click():
     }
     notification = get_mock_notification(message)
     return mail, click, notification
+
+
+def get_mock_received_s3():
```

Comment:
Looks like we need a test for this, too

Reply:
I think tests for these mocks are needed still, I guess to cover each of the handler classes now!